wgs/PairedEndSingleSampleWf/README.md

```diff
@@ -0,0 +1,7 @@
+The workflow in this repository is the Paired End Single Sample workflow provided by the Broad Institute.
+
+A copy of the PairedEndSingleSampleWf.wdl file was made on January 4, 2018 from the Broad's github repo:
```

Comment:
> A copy of the January 4, 2018 version of the `PairedEndSingleSampleWf.wdl` file was made from the Broad Institute's github repo:

(I'm not actually sure when we made the copy - just know it was done after 1/4/2018).

Reply:
This change has been made